x/programs/simulator/src/lib.rs

```diff
@@ -1,12 +1,35 @@
-#![allow(non_upper_case_globals)]
-#![allow(non_camel_case_types)]
-#![allow(non_snake_case)]
-// include the generated bindings
-// reference: https://rust-lang.github.io/rust-bindgen/tutorial-3.html
-include!(concat!(env!("OUT_DIR"), "/bindings.rs"));
+mod bindings {
+    #![allow(non_upper_case_globals)]
+    #![allow(non_camel_case_types)]
+    #![allow(non_snake_case)]
+    #![allow(dead_code)]
+
+    use wasmlanche_sdk::Address as SdkAddress;
+    // include the generated bindings
+    // reference: https://rust-lang.github.io/rust-bindgen/tutorial-3.html
+    include!(concat!(env!("OUT_DIR"), "/bindings.rs"));
```

Comment:
This should be cleaned up to use `PathBuff::new(env!("OUT_DIR"))` and `join` instead of adding `/` by hand.